GitHubLabels.ps1

```diff
@@ -565,6 +628,121 @@ function Set-GitHubLabel
     }
 }
 
+function Add-GitHubLabel
+{
+<#
+    .DESCRIPTION
+        Adds a label to an issue in the given GitHub repository.
```

Comment:
Looks like you're missing the [Replace all labels for an issue](https://developer.github.com/v3/issues/labels/#replace-all-labels-for-an-issue) functionality.